semi_1/variable.ts

```diff
@@ -0,0 +1,67 @@
+// string
+const test1: string = 'hello'
+console.log(typeof test1, test1) 
+
+// boolean
+const test3: boolean = true
+console.log(typeof test3, test3)
+
+// array 표현방법
+let numbers: number[]  = [1,2,3];
+let hangeul: Array<String> = ["가", "나", "다"];
+
+// object
+const foo1= (a: Object) =>{
+    console.log(a)
+}
+
+const foo2 = (a: object) => {
+    console.log(a)
+}
+
+foo1('hello')
+// foo2('hello')
+
+// function
+const h = (name: string): void => { // void : 리턴값
+    console.log(`${name}아 안녕`)
+}
+
+const sum1 = (a: number, b:number): number => {
+    return a+b;
+}
+
+h("예원");
+sum1(1,2);
+
+// * as
+const test11: any = '소예원' // any는 타입스크립트 사용 시 지양
+const nameLength = (test11 as string).length
+console.log(nameLength)
+
+const test12: any = '장히리'
+const nameLength2 = (<string>test12).length
+console.log(nameLength2)
+ 
+// Interface
+interface TEST 
```

Comment:
Interface 네이밍 컨벤션은 Pascal Case를 권장합니다!

세미나 때도 중간에 이야기하면서 바꿨었는데, 아래와 같이 바꾸시면 좋을 것 같아요!
```suggestion
interface Test
```

Reply:
앗 알려주셔서 감사합니다!!!